lib/logstash/outputs/influxdb.rb

```diff
@@ -20,7 +20,10 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   config_name "influxdb"
 
   # The database to write
-  config :db, :validate => :string, :default => "stats"
+  config :db, :validate => :string, :default => "statistics"
```

Comment:
"stats" is a reserved word in Influxdb 0.9 and using it results in errors. You are correct that this is backwards-incompatible.